### PR TITLE
DOT-1444 language sync to wiki

### PR DESCRIPTION
**Pull request by edwh**
As discussed, a cheap and cheerful command to sync the languages.

### CODE REVIEW

File: app/Providers/ScheduleServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+use Illuminate\Console\Scheduling\Schedule;
+
+class ScheduleServiceProvider extends ServiceProvider
```
Comment (ngm) [inline app/Providers/ScheduleServiceProvider.php:8]: Is this auto-registered somehow?

**Review by ngm — APPROVED** [review]
Looks good to me - just wondering how we ScheduleServiceProvider is registered (I don't remember how this works, so perhaps it's just automagically registered by virtue of being in the right folder)

Also - does it need a cron job to cause the schedule to run every 5 minutes, or Laravel schedules it itself?

File: app/Providers/ScheduleServiceProvider.php (hunk 1 above)
Comment (edwh) [reply app/Providers/ScheduleServiceProvider.php:8]: No, it's registerd in app.php around line 173.

**Comment by edwh** [comment]
> Also - does it need a cron job to cause the schedule to run every 5 minutes, or Laravel schedules it itself?

It does need a cron job.  When I added a scheduled task to the Repair Directory I added this on production:

`* * * * * cd /srv/users/serverpilot/apps/repairdirectory && php artisan schedule:run >> /tmp/laravel_schedule.out 2>&1`

So we'd do something similar, and tweak the output file name.

File: app/Providers/ScheduleServiceProvider.php (hunk 1 above)
Comment (ngm) [reply app/Providers/ScheduleServiceProvider.php:8]: Ah right, yes, missed that somehow :+1:

**Pull request merged**